docs/enhancements/on-prem-data-collection.md

```diff
@@ -0,0 +1,121 @@
+---
+title: on-prem-data-collection
+authors:
+  - "@CrystalChun"
+creation-date: 2022-09-30
+last-updated: 2022-09-30
+---
+
+# On-premises Data Collection
+
+## Summary
+
+To provide a better installation process and experience, the Assisted Installer ("AI") will collect data about installations when it is deployed on-premises as part of either [Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6) ("RHACM") or as a stand-alone Podman deployment. Data was already collected from clusters deployed on [Red Hat's Hybrid Cloud Console](https://console.redhat.com/openshift/assisted-installer/clusters) (“SaaS”). The collection of data from on-premises deployments will give the team additional insight into the product's usage, customers' pain points, and analytics for how to best improve the product.
+
+## Motivation
+
+Data helps us see how the product is being used, helps us to analyze trends, and to determine ways we can improve the product for our users. 
+
+### Goals
+
+1. Create a process to collect and send [event](https://github.com/openshift/assisted-service/tree/master/docs/dev/events.md) data from on-premises deployments of AI.
+2. Analyze and process on-premises data (by use of visualizations, dashboards, etc.) at Red Hat like SaaS data.
+
+
+### Non-Goals
+
+1. Implementing new data types to collect is out of the scope of this epic. This epic defines how and when the data already collected will be processed and sent.
```

Comment:
Thanks for bringing up this point! I'm not so sure data formatting should be handled in this epic, but it would definitely affect this epic. 

Is there already a discussion around this somewhere? Do you happen to have a link?

Reply:
No official discussion yet I am afraid... We can start one of course :)

Reply:
Good to know! Where would you recommend we start discussing this?

Reply:
I have started a rough document with what I have in mind, maybe we can start the discussion there and see where it goes?  Should it eventually land in this very enhancement or a specific one? https://docs.google.com/document/d/1MUtjnhhGxKvY1pjnOV4caFR6FnO_5_qjrAHDhJs-Nrw

Reply:
Awesome, thanks Riccardo! Maybe it could be its own? 

It's related to this epic, but I think restructuring the data might be its own effort. WDYT?

Reply:
Sounds good to me!